features/messages/impl/src/main/kotlin/io/element/android/features/messages/impl/MessagesPresenter.kt

```diff
@@ -121,6 +123,8 @@ class MessagesPresenter @AssistedInject constructor(
 
     @Composable
     override fun present(): MessagesState {
+        htmlConverterProvider.Update()
```

Comment:
Do we really want this to be called each time the state changes?

Reply:
I think I can probably make it skippable so it's only run when needed.

Reply:
Actually, it's not possible because it contains a couple of composable functions that return values and those can't be skipped, so neither can their parents. However, I can make sure they're stable and the 'heavy work' is only done when some value changes.